packages/preview/acrotastic/0.1.0/typst.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name="acrotastic"
+version="0.1.0"
+entrypoint="acrotastic.typ"
+authors=["@Julian702"]
+license="MIT"
+description="Manage acronyms and their definitions in Typst."
+repository="https://github.com/Julian702/typst-packages"
+keywords=["acronym", "acronyms", "acro", "abbreviation", "abbreviations", "abbrev"]
+categories=["model", "paper", "thesis"]
```

Comment:
I think just "model" is enough. "paper" and "thesis" are a bit too unrelated.

Reply:
Hey @laurmaedje ,
thanks for your feedback. Based on my own experience, I assumed that a lot of papers or theses have a list of abbreviations. That's why I thought it would be helpful to find the package there when writing a scientific paper. I've removed the categories for now, but can add them back if you like :)

Reply:
I see how that could also be useful, but so far the paper & thesis categories were intended to mostly contain things that are truly specific to those (mostly templates). So I think it's better to keep the change. :)